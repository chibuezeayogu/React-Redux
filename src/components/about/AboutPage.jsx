import React from 'react';

const AboutPage = () => (
	<div className="jumbotron">
		<h1 className="display-4">About</h1>
		<p className="lead">This is about page!.</p>
		<hr className="my-4"/>
	</div>
);

export default AboutPage;
