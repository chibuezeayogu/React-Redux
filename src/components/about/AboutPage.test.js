import React from 'react';
import { shallow } from 'enzyme';

import AboutPage from './AboutPage';

describe('AboutPage Component', () => {
	const wrapper = shallow(<AboutPage />);

	it('should render correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
