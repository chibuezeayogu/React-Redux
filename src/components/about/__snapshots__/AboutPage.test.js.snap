// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Component should render correctly 1`] = `
<div
  className="jumbotron"
>
  <h1
    className="display-4"
  >
    About
  </h1>
  <p
    className="lead"
  >
    This is about page!.
  </p>
  <hr
    className="my-4"
  />
</div>
`;
